apps/frontend/src/libs/components/breadcrumbs/breadcrumbs.tsx

```diff
@@ -0,0 +1,78 @@
+import { Icon } from "~/libs/components/components.js";
+import { useLocation } from "~/libs/hooks/hooks.js";
+
+import styles from "./styles.module.css";
+
+type BreadcrumbItem = {
+	link?: string;
+	name: string;
+};
+
+type Properties = {
+	items?: BreadcrumbItem[];
+};
+
+const FIRST_ITEM_INDEX = 0;
+const SINGLE_ITEM_OFFSET = 1;
+
+const formatBreadcrumbName = (name: string): string => {
+	return (
+		name
+			.replaceAll("-", " ")
+			// eslint-disable-next-line unicorn/prefer-string-replace-all
+			.replace(/\b\w/g, (char) => char.toUpperCase())
+	);
+};
```

Comment:
It looks like a helper

Reply:
Yep, will be creating a seperate file for this one

Reply:
agree, better move this to helpers